lib/tasks/application.rake

```diff
@@ -121,4 +121,31 @@ namespace :application do
     Rails.logger = ActiveSupport::Logger.new($stdout)
     Rails.logger.level = 1
   end
+
+  namespace :links_valid do
+    desc "Checks that all links in markdown for divisions are valid"
```

Comment:
The description doesn't actually accurately reflect what this is doing

Reply:
Updated description in the last commit.